app/src/main/java/crux/bphc/cms/fragments/CourseEnrolFragment.kt

```diff
@@ -130,7 +160,7 @@ class CourseEnrolFragment : Fragment() {
                         .replace(R.id.course_section_enrol_container, courseSectionFragment)
                         .commit()
                 } else {
-                    Toast.makeText(activity, "Unknown error occurred", Toast.LENGTH_SHORT).show()
+                    Toast.makeText(activity, "Course Already Enrolled", Toast.LENGTH_SHORT).show()
```

Comment:
Consider rephrasing it to something like "You're already enrolled in this course"?

Reply:
And make it a string resource while you're at it.

Reply:
Rephrased the message to "You are already enrolled in this course" and made a string resource named "course_enrol_course_already_enrolled_msg" under the section of values for "Course Enrol".
I have also changed the commit message and squashed the commits.